eth2/operation_pool/src/lib.rs

```diff
@@ -75,6 +75,9 @@ impl<T: EthSpec> OperationPool<T> {
         state: &BeaconState<T>,
         spec: &ChainSpec,
     ) -> Result<(), AttestationValidationError> {
+        // Check that attestation signatures are valid.
+        verify_attestation_for_state(state, &attestation, VerifySignatures::True, spec)?;
```

Comment:
Is this check necessary? It seems to be a duplicate of the `BeaconChain` check:

https://github.com/sigp/lighthouse/blob/fast-sig-verify/beacon_node/beacon_chain/src/beacon_chain.rs#L727-L747

Reply:
Oh great find!